lib/devpack.rb

```diff
@@ -177,9 +179,13 @@ def unpack(archive, target_dir, includes = [])
     puts "extracting '#{archive}' to '#{target_dir}'"
     case File.extname(archive)
     when '.zip', '.7z', '.exe'
-      system("\"#{zip_exe}\" x -o\"#{target_dir}\" -y \"#{archive}\" -r #{includes.join(' ')} 1> NUL")
+      begin
+        sh("\"#{zip_exe}\" x -o\"#{target_dir}\" -y \"#{archive}\" -r #{includes.join(' ')} 1> NUL")
+      rescue
+        raise "Upacking failed with #{$!.message}.\nPerhaps 7-zip is not on the PATH or ZIP_EXE is not defined?" 
+      end
     when '.msi'
-      system("start /wait msiexec /a \"#{archive.gsub('/', '\\')}\" /qb TARGETDIR=\"#{target_dir.gsub('/', '\\')}\"")
+      sh("start /wait msiexec /a \"#{archive.gsub('/', '\\')}\" /qb TARGETDIR=\"#{target_dir.gsub('/', '\\')}\" INSTALLDIR=TARGETDIR=\"#{target_dir.gsub('/', '\\')}\" APPDIR=TARGETDIR=\"#{target_dir.gsub('/', '\\')}\"")
```

Comment:
oh no, I meant the `INSTALLDIR=TARGETDIR=...` and `APPDIR=TARGETDIR=...` pieces

Guess the `TARGETDIR=` is too much then...